drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -367,6 +360,53 @@ RigidBodySpringDamper::RigidBodySpringDamper(RigidBodySystem* sys,
   tree->addFrame(frameB);
 }
 
+RigidBodyAccelerometer::RigidBodyAccelerometer(RigidBodySystem *sys,
+                                               const std::string& name,
+                                               const std::shared_ptr<RigidBodyFrame> frame)
+  : RigidBodySensor(sys,name), frame(frame) {  }
+
+Eigen::VectorXd RigidBodyAccelerometer::output(const double &t,
+                                               const KinematicsCache<double> &rigid_body_state,
+                                               const RigidBodySystem::InputVector<double>& u) const
+{
+  auto q = rigid_body_state.getQ();
+  auto v = rigid_body_state.getV();
+  VectorXd x(q.size() + v.size());
+  x << q, v;
+  auto xdd = sys->dynamics(t, x, u);
+  auto tree = sys->getRigidBodyTree();
+  auto v_dot = xdd.bottomRows(v.size());
+  Vector3d sensor_origin = Vector3d::Zero(); //assumes sensor coincides with the frame's origin;
```

Comment:
> Using the GPF?

We're just using a slightly modified canonical scan matcher for now.  We did contemplate using a known map with GPF localization for high speed flights but the scan matcher has been fine so far.